### PR TITLE
Use idp logo from theme

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/11603

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: services/idp/src/components/ResponsiveScreen.jsx [hunk 1]
```diff
@@ -45,8 +45,26 @@ const ResponsiveScreen = (props) => {
...
   } = props;
 
-  const logo = withoutLogo ? null :
-    <img src={process.env.PUBLIC_URL + '/static/logo.svg'} className="oc-logo" alt="ownCloud Logo"/>;
+    const [theme, setTheme] = useState(null);
+
+    useEffect(() => {
+        const fetchTheme = async () => {
+            try {
+                const response = await fetch('/themes/owncloud/theme.json');
```
[the remark this answers is not in the document]
Comment (AlexAndBear) [reply services/idp/src/components/ResponsiveScreen.jsx]: ok loading path from config now

File: services/idp/src/App.jsx [hunk 2]
```diff
@@ -1,31 +1,64 @@
...
-      className='oc-login-bg'
-      style={{ backgroundImage: bgImg ? `url(${bgImg})` : undefined }}
-    >
-      <MuiThemeProvider theme={theme}>
-        <Suspense fallback={<Spinner/>}>
-          <LazyMain/>
-        </Suspense>
-      </MuiThemeProvider>
-    </div>
+      <InfiniteScaleContext.Provider value={{ theme: theme, config: config }}>
```
Comment (AlexAndBear) [inline services/idp/src/App.jsx:50]:
```suggestion
      <InfiniteScaleContext.Provider value={{ theme, config }}>
```

**Review by JammingBen — COMMENTED** [review]
On a fresh instance (remove `.ocis` folder and re-init) it tries to fetch the default logo from `themes/owncloud/assets/logo.svg`, which doesn't exist. Does it happen for you as well, or just on my end?

Edit: ah, I assume it's only happening to me because of the web config file I'm using.

**Review by JammingBen — APPROVED** [review]
Clarified it in a call, the issue was that I had my own web dist folder referenced.

LGTM, nice 🚀

**Pull request merged**